src/sentry/sentry_metrics/configuration.py

```diff
@@ -10,19 +10,31 @@ class UseCaseKey(Enum):
     PERFORMANCE = "performance"
 
 
+# Rate limiter namespaces, the postgres (PG)
+# values are the same as UseCaseKey to keep
+# backwards compatibility
+RELEASE_HEALTH_PG_NAMESPACE = "release-health"
+PERFORMANCE_PG_NAMESPACE = "performance"
+
+
 @dataclass(frozen=True)
 class MetricsIngestConfiguration:
     input_topic: str
     output_topic: str
     use_case_id: UseCaseKey
     internal_metrics_tag: Optional[str]
     writes_limiter_cluster_options: Mapping[str, Any]
+    writes_limiter_namespace: str
 
 
 _METRICS_INGEST_CONFIG_BY_USE_CASE: MutableMapping[UseCaseKey, MetricsIngestConfiguration] = dict()
 
 
 def _register_ingest_config(config: MetricsIngestConfiguration) -> None:
+    for registered_config in _METRICS_INGEST_CONFIG_BY_USE_CASE.values():
+        assert (
+            registered_config.writes_limiter_namespace != config.writes_limiter_namespace
+        ), "namespace must be unique!"
```

Comment:
Is this needed? I'd prefer it was done in a test rather than at runtime.

Reply:
yeah just changed the test do the unique check